src/validation.h

```diff
@@ -83,6 +84,15 @@ static const int DEFAULT_STOPATHEIGHT = 0;
 static const unsigned int MIN_BLOCKS_TO_KEEP = 288;
 static const signed int DEFAULT_CHECKBLOCKS = 6;
 static const unsigned int DEFAULT_CHECKLEVEL = 3;
+static const std::vector<std::string> CHECKLEVEL_DOC = {
+    "level 0 reads the blocks from disk",
+    "level 1 verifies block validity",
+    "level 2 verifies undo data",
+    "level 3 checks disconnection of tip blocks",
+    "level 4 tries to reconnect the blocks",
+    "each level includes the checks of the previous levels"
```

Comment:
```suggestion
    "each level includes the checks of the previous levels",
```

Nit: Add a trailing comma for consistency and to make clang-format happy?

Reply:
Addressed